contracts/Insurance.sol

```diff
@@ -121,10 +123,32 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
      * @dev Withdraws from tracer, and adds amount to the pool's amount field.
      */
     function updatePoolAmount() public override {
-        int256 quote = (tracer.getBalance(address(this))).position.quote;
-        if (quote > 0) {
-            tracer.withdraw(uint256(quote));
-            collateralAmount = collateralAmount + uint256(quote);
+        uint256 quote =
+            uint256((tracer.getBalance(address(this))).position.quote);
+
+        tracer.withdraw(quote);
+
+        if (publicCollateralAmount > 0) {
```

Comment:
Just confirming this is correct, and shouldn't be `bufferCollateralAmount`.

Reply:
Yes; only when someone has deposited into the public insurance account do funds start accruing proportionally to the buffer and public pools; otherwise, they all go to the public pool